src/lib.rs

```diff
@@ -238,25 +254,24 @@ impl<'a> Server<'a> {
     }
 }
 
-pub fn download(remote_ssh_host : &str,
-                port_range      : connection::PortRange,
-                remote_path     : &str,
-                local_path      : PathBuf)
-{
+pub fn download(remote_ssh_host: &str,
+                port_range: connection::PortRange,
+                remote_path: &str,
+                local_path: PathBuf) {
```

Comment:
i need to find a way to tell rustfmt to align more like the previous

moxie has ruined me with this now any other way looks weird :smile:

but yeah, let's start with a basepoint of rustfmt-ted code and then i can whine about style and try to play with the fmt config later.